test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/DthStartupTests.cs

```diff
@@ -308,24 +295,24 @@ public void DthNegative_BrokenProjectPathInLockFile_V1(DnxSdk sdk)
         [MemberData(nameof(DnxSdks))]
         public void DthNegative_BrokenProjectPathInLockFile_V2(DnxSdk sdk)
         {
-            var testProject = _fixture.GetTestProjectPath("BrokenProjectPathSample");
+            var projectName = "BrokenProjectPathSample";
+            var project = GetTestProject(sdk, projectName).Restore(sdk, ensureSuccess: false);
 
             using (var disposableDir = new DisposableDir())
             using (var server = DthTestServer.Create(sdk))
             using (var client = server.CreateClient())
             {
-                // copy the project to difference location so that the project path in its lock file is invalid
-                var targetPath = Path.Combine(disposableDir, "BrokenProjectPathSample");
-                Testing.TestUtils.CopyFolder(testProject, targetPath);
+                var movedProjectPath = Path.Combine(disposableDir, projectName);
+                TestUtils.CopyFolder(project.ProjectDirectory, movedProjectPath);
```

Comment:
Because 1. Editing the json file is move complex; 2. Once we update the lock file format the test needs to be rebaseline.

Here actually only two files are copied and it mimic the real situation when the bug was found.